tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/Lambda/LambdaCommon.cs

```diff
@@ -58,15 +67,42 @@ internal static Scope SendStartInvocation(ILambdaExtensionRequest requestBuilder
         WriteRequestHeaders(request, context);
         var response = (HttpWebResponse)request.GetResponse();
         var traceId = response.Headers.Get(HttpHeaderNames.TraceId);
+        var traceIdUpper64 = GetTraceIdUpper64(response.Headers.Get(HttpHeaderNames.PropagatedTags));
         var samplingPriority = response.Headers.Get(HttpHeaderNames.SamplingPriority);
```

Comment:
This code (and `GetTraceIdUpper64()` below) only support the Datadog-style headers (`x-datadog-trace-id`, `x-datadog-sampling-priority`, etc) and is duplication header extraction code. We should use `SpanContextPropagator.Instance.Extract()` instead.